product_category_active/tests/test_product_category_active.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import SavepointCase
+
+
+class TestProductCategoryActive(SavepointCase):
```

Comment:
as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0:

Replace SavepointCase by TransactionCase in tests, as they are now the same. The old one still exists as an alias, but a warning will arise, and next version will remove such alias.